the-experts-BE/app.js

```diff
@@ -7,6 +7,7 @@ var logger = require('morgan');
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
 var testAPIRouter = require("./routes/testAPI");
+var getContractor=require("./routes/contractor-results");
```

Comment:
Why are you using `var` for all these variables? You are not mutating their values, you should be using `const`.